### PR TITLE
Fix installation steps in the doc tests section

### PR DESCRIPTION
**Pull request by martinRenou**

### CODE REVIEW

File: docs/source/contribute.rst [hunk 1]
```diff
@@ -143,13 +143,13 @@ Install the test dependencies
 
 .. code-block:: bash
 
-   python -m pip install -e ".[test]"
+   python -m pip install ".[test]"
```
Comment (maartenbreddels) [inline docs/source/contribute.rst:146]: We can leave the -e if we enable the extension right?

Comment (martinRenou) [reply docs/source/contribute.rst:146]: Maybe? I was not able to run any test on my side. I did not find a combination that works.

Comment (maartenbreddels) [reply docs/source/contribute.rst:146]: Did you run `(cd tests/test_template; pip install .)` ?

Comment (martinRenou) [reply docs/source/contribute.rst:146]: yep

**Pull request merged**